po/de.po

```diff
@@ -17,46 +17,52 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: src/menu/menu.c:631
+#: src/menu/menu.c:698
 msgid "Reconfigure"
 msgstr "Rekonfigurieren"
 
-#: src/menu/menu.c:633
+#: src/menu/menu.c:700
 msgid "Exit"
 msgstr "Beenden"
 
-#: src/menu/menu.c:649
+#: src/menu/menu.c:716
 msgid "Minimize"
 msgstr "Minimieren"
 
-#: src/menu/menu.c:651
+#: src/menu/menu.c:718
 msgid "Maximize"
 msgstr "Maximieren"
 
-#: src/menu/menu.c:653
+#: src/menu/menu.c:720
 msgid "Fullscreen"
 msgstr "Vollbild"
 
-#: src/menu/menu.c:655
+#: src/menu/menu.c:722
 msgid "Decorations"
 msgstr "Dekorationen"
 
-#: src/menu/menu.c:657
-msgid "Always On Top"
+#: src/menu/menu.c:724
+#, fuzzy
+#| msgid "Always On Top"
+msgid "Always on Top"
```

Comment:
Oh, it appears we have used `On` in the `.po` files but `on` (lowercase) as actual msgid and in the C sources.
We should likely fix that.

Reply:
Good catch!

Reply:
> Oh, it appears we have used `On` in the `.po` files but `on` (lowercase) as actual msgid and in the C sources. We should likely fix that before 0.7.0.

Looks like the bot has caught that! (see latest force push)

Reply:
Looks like the bot has fixed this now :smile: